app/views/eventDetails/regTypes.html

```diff
@@ -138,7 +138,7 @@ <h4 translate>Payment</h4>
                 </i>
               </div>
 
-              <div class="pt12 input-group" ng-if="conference.requireLogin && anyPaymentMethodAccepted(type)">
+              <div class="pt12 input-group" ng-if="(conference.relayLogin || conference.facebookLogin) && anyPaymentMethodAccepted(type)">
```

Comment:
It may be a good idea to move this check `conference.relayLogin || conference.facebookLogin` into an attribute on conference.